terminus-ui/chip/src/chip-collection.component.ts

```diff
@@ -222,41 +224,27 @@ export class TsChipCollectionComponent implements AfterViewInit, OnInit, OnDestr
   /**
    * The chip components contained within this chip collection.
    */
-  @ViewChildren(TsChipComponent) public chips!: QueryList<TsChipComponent>;
-
-  /**
-   * A list of chip values as input.
-   */
-  @Input()
-  public chipsInput: ReadonlyArray<string> = [];
-
-  @Input()
-  public displayFormatter = v => v as string
+  @ContentChildren(TsChipComponent)
+  public chips!: QueryList<TsChipComponent>;
 
   /**
    * Whether the user should be allowed to select multiple chips.
    */
   @Input()
   public set allowMultipleSelections(value: boolean) {
-    this._multiple = coerceBooleanProperty(value);
+    this._allowMultipleSelections = value;
     this.syncChipsState();
   }
   public get allowMultipleSelections(): boolean {
-    return this._multiple;
+    return this._allowMultipleSelections;
   }
-  private _multiple = false;
+  private _allowMultipleSelections = false;
 
   /**
-   * Set and get chip collection value
+   * Orientation of the chip collection.
    */
-  @Input()
-  public set value(value: string[]) {
-    this._value = value;
-  }
-  public get value(): string[] {
-    return this._value;
-  }
-  protected _value = [''];
+  @Input('aria-orientation')
+  public ariaOrientation: 'horizontal' | 'vertical' = 'horizontal';
```

Comment:
Git seems a bit confused on some reorganization in this and the chip files - so be aware that many items were just moved rather than deleted.